src/Config/Config.ts

```diff
@@ -173,11 +173,12 @@ export default class Config implements IConfigParams {
       });
   }
 
-  private _economicStrategyToBN(economicStrategy: IEconomicStrategy) {
+  private _economicStrategyToBN(economicStrategy: any) {
+    const ethToBN = (num: number) => new BigNumber(this.web3.toWei(num, 'ether'));
     return {
-      maxDeposit: new BigNumber(economicStrategy.maxDeposit),
-      minBalance: new BigNumber(economicStrategy.minBalance),
-      minProfitability: new BigNumber(economicStrategy.minProfitability),
+      maxDeposit: ethToBN(economicStrategy.maxDeposit),
```

Comment:
Why have we switch to ETH from WEI? I think this conversion should be done in UI only for sake of readability.

Reply:
This is only used internally to convert the default settings https://github.com/ethereum-alarm-clock/timenode-core/blob/master/src/Config/Config.ts#L16.

The default settings are exported along with the Config object and made available. I was reluctant to export the defaults as BigNumber cause different implementations use different BN libs (or none at all). But I guess I could export 1000000000000000000 instead of 1, and then . just convert it to a BN locally.

Reply:
Config.DEFAULT_ECONOMIC_STRATEGY is used by the CLI to show the default values in `--help`.